ios/Classes/Blemulator/Bridging/Constants/SimulationChannelName.m

```diff
@@ -0,0 +1,4 @@
+#define BASE @"com.polidea.blemulator"
+
+NSString * const SIMULATION_CHANNEL_NAME_TO_DART = (BASE @"/toDart");
+NSString * const SIMULATION_CHANNEL_NAME_TO_OBJC = (BASE @"/toObjC");
```

Comment:
Nu-uh, it's the same channel as the one used in Java, so either `/toJava` or rename both to `/toPlatform`. The latter sounds better.

Reply:
I like `/toPlatform`